argo-cd-apps/overlays/development/kustomization.yaml

```diff
@@ -101,6 +101,11 @@ patches:
       kind: ApplicationSet
       version: v1alpha1
       name: release
+  - path: development-overlay-patch.yaml
+    target:
+      kind: ApplicationSet
+      version: v1alpha1
+      name: mintmaker
```

Comment:
this patch is setting the environment to `development` in the ApplicationSet which is what we want. You need to do the same in but using `production-overlay-patch.yaml`:
- argo-cd-apps/overlays/konflux-public-production/kustomization.yaml
- argo-cd-apps/overlays/production-downstream/kustomization.yaml